core/conn/jdbc_type2/native/CSrvrStmt.cpp

```diff
@@ -341,7 +479,7 @@ SQLRETURN SRVR_STMT_HDL::Close(unsigned short inFreeResourceOpt)
     SQLRETURN rc;
 
     if (stmtType == INTERNAL_STMT) CLI_DEBUG_RETURN_SQL(SQL_SUCCESS);
-    cleanupSQLMessage();
+    //cleanupSQLMessage();
```

Comment:
Sorry, this is not on purpose, I will uncomment this within an updates along with some other changes of the memory allocation/release management code.